collect_app/src/androidTest/java/org/odk/collect/android/feature/formentry/FormCalculationsTest.kt

```diff
@@ -0,0 +1,69 @@
+package org.odk.collect.android.feature.formentry
+
+import android.widget.FrameLayout
+import android.widget.TextView
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import junit.framework.TestCase.assertNotSame
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.RuleChain
+import org.junit.runner.RunWith
+import org.odk.collect.android.R
+import org.odk.collect.android.activities.FormFillingActivity
+import org.odk.collect.android.support.ActivityHelpers
+import org.odk.collect.android.support.pages.MainMenuPage
+import org.odk.collect.android.support.rules.CollectTestRule
+import org.odk.collect.android.support.rules.TestRuleChain
+
+@RunWith(AndroidJUnit4::class)
+class FormCalculationsTest {
+    private val rule = CollectTestRule()
+
+    @get:Rule
+    val ruleChain: RuleChain = TestRuleChain.chain().around(rule)
+
+    @Test
+    fun randomFunction_ShouldWorkCorrectly() {
```

Comment:
This feels like it should be covered in JavaRosa? Does Collect actually implement anything to do with `random()`?

Reply:
> Does Collect actually implement anything to do with random()?

I don't think so. Ok we can gett rid of this.